### PR TITLE
new whiteboard file

### PR DESCRIPTION
**Pull request by skannan1985**
pls review annie

### CODE REVIEW

File: src/whiteboard [hunk 1]
```diff
@@ -0,0 +1,5 @@
+you 
+can
+write
+on
+whiteboard
```
Comment (sujanyar) [inline src/whiteboard:5]: write using what?

**Review by sujanyar — CHANGES_REQUESTED** [review]

File: src/whiteboard (hunk 1 above)
Comment (skannan1985) [reply src/whiteboard:5]: added details

File: src/whiteboard [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+using marker
```
Comment (sujanyar) [inline src/whiteboard:6]: And then one more name?

**Review by sujanyar — CHANGES_REQUESTED** [review]

File: src/whiteboard (hunk 2 above)
Comment (skannan1985) [reply src/whiteboard:6]: one more name added

**Review by sujanyar — APPROVED** [review]

**Review by sujanyar — APPROVED** [review]

**Pull request merged**